pkg/handlers/v1/notification.go

```diff
@@ -65,6 +69,7 @@ func (h *NotificationHandler) Handle(ctx context.Context) (Output, error) {
 		if err := h.TimestampStorer.StoreTimestamp(ctx, scan.Timestamp); err != nil {
 			return Output{}, err
 		}
+		stater.Timing("scannotificationdelay", time.Since(scanCompletedTime)*time.Millisecond)
```

Comment:
task: The result of `time.Since()` should be used directly here and not multiplied by `time.Millisecond`. The output for `time.Since()` is a `time.Duration` type which does not need to be converted. In fact, multiplying my `time.Millisecond` here actually does a bad thing and breaks the value you want. Here's a playground to illustrate: https://play.golang.org/p/e7bzLBty_vB

Reply:
done!

Reply:
done!